cloud/logger/src/EmptyLogger.ts

```diff
@@ -0,0 +1,19 @@
+import {Logger} from './Logger';
+
+export class EmptyLogger implements Logger {
```

Comment:
@paolobueno I remember that we can derrive TS interface from class. 
WDYT about doing that here to reduce code duplication?